src/ui_ng/lib/src/list-replication-job/list-replication-job.component.html.ts

```diff
@@ -0,0 +1,25 @@
+export const REPLICATION_JOB_TEMPLATE: string = `
+<clr-datagrid (clrDgRefresh)="refresh($event)">
+    <clr-dg-column>{{'REPLICATION.NAME' | translate}}</clr-dg-column>
+    <clr-dg-column>{{'REPLICATION.STATUS' | translate}}</clr-dg-column>
+    <clr-dg-column>{{'REPLICATION.OPERATION' | translate}}</clr-dg-column>
+    <clr-dg-column>{{'REPLICATION.CREATION_TIME' | translate}}</clr-dg-column>
+    <clr-dg-column>{{'REPLICATION.END_TIME' | translate}}</clr-dg-column>
+    <clr-dg-column>{{'REPLICATION.LOGS' | translate}}</clr-dg-column>
+    <clr-dg-row *clrDgItems="let j of jobs" [clrDgItem]='j'>
+        <clr-dg-cell>{{j.repository}}</clr-dg-cell>
+        <clr-dg-cell>{{j.status}}</clr-dg-cell>
+        <clr-dg-cell>{{j.operation}}</clr-dg-cell>
+        <clr-dg-cell>{{j.creation_time}}</clr-dg-cell>
+        <clr-dg-cell>{{j.update_time}}</clr-dg-cell>
+        <clr-dg-cell>
+            <a href="/api/jobs/replication/{{j.id}}/log" target="_BLANK">
+                <clr-icon shape="clipboard"></clr-icon>
+            </a>
+        </clr-dg-cell>
+    </clr-dg-row>
+    <clr-dg-footer>
+        {{ jobs ? jobs.length : 0 }} {{'REPLICATION.ITEMS' | translate}}
```

Comment:
We should use the same pagination style here like others do.

{{pagination.firstItem + 1}} - {{pagination.lastItem + 1}} of {{pagination.totalItems}} users
        <clr-dg-pagination #pagination [clrDgPageSize]="15" [clrDgTotalItems]="users.length"></clr-dg-pagination>